### ISSUE TITLE
Search field not working in Safari

### ISSUE THREAD

**Issue by wout** [wolffe/tail.select.js#3]
In the latest Safari (Version 14.1 - 16611.1.21.161.3), the search field doesn't work. You can enter one character, but then it's somehow blocked:

<img width="529" alt="image" src="https://user-images.githubusercontent.com/107324/116992980-bf709900-acce-11eb-9179-0922f2a7d528.png">

In other browsers it's ok.

**Comment by wolffe** [comment]
I have no way of testing this with Safari. Any contributions are welcome, if you can find the issue.

**Comment by wout** [comment]
Thanks for your quick reply. I don't have the time really to take a closer look at it. We're using it for an internal app, so people now simply use a different browser. Hopefully, someone else can find the time to look into it. It's a great library.

**Comment by wolffe** [comment]
I know it's a great library, that's why I revived it. I actually have a coworker using a Mac, so I'll ask him to test it thoroughly, maybe I can find the issue.

I'll leave this open, as I'll look into it as some point, or maybe you'll find some time to take a look. Maybe the focus is changing automatically somehow or another layer is being moved forward.

**Comment by hoanguyenthai** [comment]
Hi, I am using this library before, I also had this problem when my colleague used Safari. I can't find the old project, glad you revived it.

So I continued searching on Google, then I found  [this answer](https://stackoverflow.com/questions/25610517/ios-devices-issues-with-html-form-input-type-text/25619579#25619579).

The root issues caused by one CSS `overrided -webkit-user-select` to `none`, so i have to prevent it by replacing this line to a CSS file or overriding it by add this line:
`
.tail-select, tail-select * {
  -webkit-user-select: initial !important;
}
`

Hope it is useful for you!  🙂
